.travis.yml

```diff
@@ -13,24 +13,15 @@ matrix:
   - os: osx
     env: PYTHON_VERSION="3.7" DEPS="numpy geos"
 
-before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-    elif  [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
-    fi;
-
 install:
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda:$HOME/miniconda/bin:$PATH"
-  - source $HOME/miniconda/etc/profile.d/conda.sh
-  - hash -r;
-  - conda update --yes conda;
-  - conda config --set changeps1 no --set restore_free_channel true;
-  - conda config --append channels conda-forge
-  - conda create -n testenv --yes python=$PYTHON_VERSION $DEPS pytest;
-  - conda activate testenv;
-  - conda --version ; python --version ; pip --version;
+  - wget https://repo.anaconda.com/pkgs/misc/conda-execs/conda-latest-$TRAVIS_OS_NAME-64.exe -O conda.exe
+  - chmod +x conda.exe
+  - export CONDA_ALWAYS_YES=1
+  - ./conda.exe config --set changeps1 no --set restore_free_channel true;
```

Comment:
Any idea if the `restore_free_channel` is (still) needed? 
Is it for the older geos versions?

Reply:
Yes, it's for the older GEOS versions

Reply:
Not so sure actually, the default channel seems to list GEOS 3.5, which is the earliest version. Testing without !